### PR TITLE
MINOR: Log firstUnstableOffset in debug log line in LogCleanerManager#cleanableOffsets

### PR DESCRIPTION
**Pull request by stanislavkozlovski**

### CODE REVIEW

File: core/src/main/scala/kafka/log/LogCleanerManager.scala [hunk 1]
```diff
@@ -561,7 +561,9 @@ private[log] object LogCleanerManager extends Logging {
       } else None
     ).flatten.min
 
-    debug(s"Finding range of cleanable offsets for log=${log.name} topicPartition=$topicPartition. Last clean offset=$lastCleanOffset now=$now => firstDirtyOffset=$firstDirtyOffset firstUncleanableOffset=$firstUncleanableDirtyOffset activeSegment.baseOffset=${log.activeSegment.baseOffset}")
+    debug(s"Finding range of cleanable offsets for log=${log.name} topicPartition=$topicPartition. Last clean offset=$lastCleanOffset" +
```
Comment (ijuma) [inline core/src/main/scala/kafka/log/LogCleanerManager.scala]: I think we're missing a space at the end of this line.

File: core/src/main/scala/kafka/log/LogCleanerManager.scala [hunk 2]
```diff
@@ -561,7 +561,9 @@ private[log] object LogCleanerManager extends Logging {
...
+      s"now=$now => firstDirtyOffset=$firstDirtyOffset firstUncleanableOffset=$firstUncleanableDirtyOffset" +
```
Comment (ijuma) [inline core/src/main/scala/kafka/log/LogCleanerManager.scala]: Looks like we're missing a space here too?

**Comment by stanislavkozlovski** [comment]
Ouch, thanks for catching @ijuma

**Comment by stanislavkozlovski** [comment]
retest this please

File: core/src/main/scala/kafka/log/LogCleanerManager.scala [hunk 3]
```diff
@@ -561,7 +561,9 @@ private[log] object LogCleanerManager extends Logging {
       } else None
     ).flatten.min
 
-    debug(s"Finding range of cleanable offsets for log=${log.name} topicPartition=$topicPartition. Last clean offset=$lastCleanOffset now=$now => firstDirtyOffset=$firstDirtyOffset firstUncleanableOffset=$firstUncleanableDirtyOffset activeSegment.baseOffset=${log.activeSegment.baseOffset}")
+    debug(s"Finding range of cleanable offsets for log=${log.name} topicPartition=$topicPartition. Last clean offset=$lastCleanOffset " +
+      s"now=$now => firstDirtyOffset=$firstDirtyOffset firstUncleanableOffset=$firstUncleanableDirtyOffset " +
```
Comment (ijuma) [inline core/src/main/scala/kafka/log/LogCleanerManager.scala:565]: Why are we using `=>` as a separator?

**Comment by ijuma** [comment]
Looks like this has a conflict.

**Pull request closed without merging**